### ISSUE TITLE
[BR]:Ineffective for sip requests

### ISSUE THREAD

**Issue by ChiMuYuan** [fail2ban/fail2ban#3652]
### Environment:

- Fail2Ban version  :
  0.11.2-6
- OS, including release name/version : 
  Ubuntu 22.04

I installed fail2ban using apt.

### The issue:

I'm a beginner, so I apologize for any mistakes.
I'm using FreeSWITCH for SIP.
When I use the command `fail2ban-client status freeswitch`, I see an IP like `149.56.252.xx` in the Banned IP list. However, this IP isn't actually banned; it can still access my FreeSWITCH. I can still see this IP performing actions in the FreeSWITCH logs. I've also observed it accessing in `sngrep`, a Ncurses SIP Messages flow viewer.
As far as I know, if an IP is in the banned list, FreeSWITCH shouldn't receive any requests from this IP

### Configuration, dump and another helpful excerpts
![image](https://github.com/fail2ban/fail2ban/assets/26971626/edf0e32c-9e54-4935-808b-3276a7b7141a)
![1703044047268](https://github.com/fail2ban/fail2ban/assets/26971626/1df4bbdd-3d74-410d-88db-3decbaf07c7a)

#### Any customizations done to /etc/fail2ban/ configuration

```
[freeswitch]

port     = 5060,5061,5066,5080,7443
action   = %(banaction)s[name=%(__name__)s-tcp, port="%(port)s", protocol="tcp", chain="%(chain)s", actname=%(banaction)s-tcp]
           %(banaction)s[name=%(__name__)s-udp, port="%(port)s", protocol="udp", chain="%(chain)s", actname=%(banaction)s-udp]
           %(mta)s-whois[name=%(__name__)s, dest="%(destemail)s"]
logpath  = /usr/local/freeswitch/log/freeswitch.log
filter   = freeswitch
maxretry = 10
banTime  = 43200
```

Thank you

**Comment by sebres** [comment]
What is actually your `banaction`? Or better provide an excerpt of dump output for `fail2ban-client --dp` for the freeswitch jail.
Depending this one should check the rules of `iptables`, `nftables` etc.

Here is a [FAQ](https://github.com/fail2ban/fail2ban/wiki/How-fail2ban-works) for self-checking (see the answer for 2nd Q).

**Comment by ChiMuYuan** [comment]
Thank for your reply.

After my check, I found that the logs in FreeSWITCH were due to a brief expiration in fail2ban.
There are no logs related to that IP during the banned list's effective period.

Abot the other issue, `sngrep` is still able to display information about that IP. 
[I found an answer in here](https://www.pbxforums.com/threads/cant-block-ip-bruteforcing-my-pbx.4017/)
`sngrep will show traffic even if it isn't making it to freeswitch. check for freeswitch responses to test if your rules are working.`

It could be because sngrep is tools at the Network layer(or Transport layer). I'm not a network engineer, so this is just my guess.
But it seems like fail2ban and iptables are working fine. Closing this issue.

Thank you again.
